frontend/model/contracts/group.js

```diff
@@ -703,13 +721,28 @@ sbp('chelonia/defineContract', {
           { username: meta.username, dateLeft: meta.createdDate },
           { meta, state, getters }
         )
+
+        meta.isRemoveOurselves = true
```

Comment:
Pls let me know if this(mutating the structure of the meta object for sending down an information to `sideEffect()` of `gi/contracts/group/removeMember`) is an issue. Or if you have an advice regarding this.

Reply:
Modifying the input data to these functions is definitely not correct. :)

In most situations it won't work anyway. The only reason why it works here is because below we immediately call the `removeMember/sideEffect`.

There is a far better solution, I'll comment elsewhere.